websites/R/RhysBOT/presence.ts

```diff
@@ -0,0 +1,38 @@
+const presence = new Presence({
+		clientId: "658765364439810048"
+	}),
+	date = Date.now();
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "Logo",
+			details: "Exploring",
+			startTimestamp: date,
+			buttons: [
+				{
+					label: "View Page",
+					url: document.location.href
+				}
+			]
+		},
+		page = document.location.pathname;
```

Comment:
No reason to name it page, please change the variable to pathname elsewhere
```suggestion
		{ pathname } = document.location;
```